packages/charts/src/components/ComposedChart/index.tsx

```diff
@@ -164,26 +166,31 @@ const ComposedChart: FC<ComposedChartProps> = forwardRef((props: ComposedChartPr
     [elements]
   );
 
-  const ComposedDataLabel = useDataLabel(
-    chartConfig.dataLabel,
-    defaults.dataLabelCustomElement,
-    defaults.dataLabelFormatter,
-    defaults.stackId
-  );
+  const ComposedDataLabel = (bar, noSizeCheck) =>
```

Comment:
I don't get the value of the bar parameter here to be honest. It seems to be always false?